src/etc/inc/dyndns.class

```diff
@@ -115,6 +116,8 @@
 	 *  DuiaDNS         - Last Tested: 25 November 2016
 	 *  DuiaDNS IPv6    - Last Tested: 25 November 2016
 	 *  Hover           - Last Tested: 15 February 2017
+	 *  DreamHost       - Not Tested
+	 *  DreamHost IPv6  - Not Tested
```

Comment:
@frankthetank We do not have access to all DynDNS providers, usually who submits a new provider is responsible to test it and make sure it works properly